test/services/chart_builder_test.rb

```diff
@@ -4,21 +4,22 @@ class ChartBuilderTest < ActiveSupport::TestCase
   test "#build_columns" do
     benchmark_run = create(:commit_benchmark_run, result: { 'some_time' => 5, 'some_other_time' => 5 })
     other_benchmark_run = create(:commit_benchmark_run, result: { 'some_time' => 5, 'some_other_time' => 5 })
-    chart_builder = ChartBuilder.new([benchmark_run, other_benchmark_run])
+    chart_builder = ChartBuilder.new([benchmark_run, other_benchmark_run], benchmark_run.benchmark_result_type)
 
-    chart_columns = chart_builder.build_columns do |benchmark_run|
-      "Commit: #{benchmark_run.initiator.sha1}"
+    chart_data = chart_builder.build_columns do |benchmark_run|
+      { commit: benchmark_run.initiator.sha1 }
     end
 
     assert_equal(
```

Comment:
Compared to massive changes introduced to `ChartBuilder`, I think it makes sense to also extend the coverage in `ChartBuilderTest`. Controller tests are great to have, but it's not enough.